Libplanet.Explorer.Executable/Program.cs

```diff
@@ -142,7 +146,19 @@ public static async Task Main(string[] args)
             await swarm.PreloadAsync(
                 dialTimeout: TimeSpan.FromSeconds(15),
                 trustedStateValidators: trustedPeers,
-                cancellationToken: cancellationToken
+                cancellationToken: cancellationToken,
+                blockDownloadFailed: (obj, args) =>
+                {
+                    foreach (var exception in args.InnerExceptions)
+                    {
+                        Log.Debug("hello world!");
+                        if (exception is InvalidGenesisBlockException invalidGenesisBlockException)
+                        {
+                            Log.Error(invalidGenesisBlockException.Stored.ToString());
+                            Log.Error(invalidGenesisBlockException.NetworkExpected.ToString());
```

Comment:
Printing out two lines of block hashes without any explanation?

Reply:
I added the lines to debug exceptions, but we can see an error message in console already. So I removed it again.